### PR TITLE
fix(accordion): styles

### PR DESCRIPTION
**Pull request by kishore03109**
## Problem

Design review for accordions, addressing changes [here](https://www.notion.so/opengov/Accordion-design-QA-7a07a952d7514f6989e8d04f61c1fac8?pvs=4).

## Solution

Fix styling issues

### CODE REVIEW

**Comment by kishore03109** [comment]
* **#1834** <a href="https://app.graphite.dev/github/pr/isomerpages/isomercms-frontend/1834?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kishore03109 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/layouts/components/Editor/components/BlockWrapper/BlockWrapper.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const BlockWrapper = ({
           backgroundColor="#055AFF"
           textColor="white"
           p="0.25rem"
+          mt="-0.75rem"
```
Comment (seaerchin) [inline src/layouts/components/Editor/components/BlockWrapper/BlockWrapper.tsx:34]: why's this required? from the [review](https://www.notion.so/opengov/Accordion-design-QA-7a07a952d7514f6989e8d04f61c1fac8?pvs=4#f499625ebfa14e94aff1662b73d6e345), this seems like accordion should have a `0.75rem` gap, not that the gap between elements should be reduced by `0.75rem`

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx [hunk 2]
```diff
@@ -17,25 +17,31 @@ export const IsomerDetailGroupView = ({
...
         >
-          <Box
-            backgroundColor={selected ? "white" : "transparent"}
-            marginTop="-0.25rem"
-            boxShadow="0px 0px 4px 0px rgba(0, 0, 0, 0.8)"
-            borderRadius="4px"
-          >
-            <Tooltip label="Edit accordion grid" hasArrow placement="top">
+          {!isProd && (
+            <Tooltip label="Edit accordion block" hasArrow placement="top">
```
Comment (seaerchin) [inline src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:44]: it feels very weird to see this here - can we cut this out and raise another PR? this seems like testing code going into production

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx [hunk 3]
```diff
@@ -112,18 +119,18 @@ export const IsomerDetailGroupView = ({
...
       padding-bottom="0.5rem"
     >
       <Box
         borderColor="base.divider.strong"
-        borderWidth="1px"
-        mt="1rem"
         h="100%"
         w="100%"
+        borderTop="1px solid #d4d4d4"
+        borderBottom="1px solid #d4d4d4"
```
Comment (seaerchin) [inline src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:133]: should we use colour tokens here

File: src/layouts/components/Editor/styles.scss [hunk 4]
```diff
@@ -193,8 +184,14 @@
...
 
+    &.is-open summary {
+      font-weight: 700;
+    }
+
     &.is-open > button::after {
-      content: url("data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2224%22%20height%3D%2224%22%20viewBox%3D%220%200%2024%2024%22%20fill%3D%22none%22%3E%3Cpath%20d%3D%22M6.29297%2013.2932L7.70697%2014.7072L12%2010.4142L16.293%2014.7072L17.707%2013.2932L12%207.58618L6.29297%2013.2932Z%22%20fill%3D%22%234372D6%22%2F%3E%3C%2Fsvg%3E");
+      font-family: "sgds-icons";
+      content: "\E93C";
+      transform: rotate(180deg);
```
Comment (seaerchin) [inline src/layouts/components/Editor/styles.scss:194]: what does this do? (specifically, what's `\E93C`)

**Review by seaerchin — COMMENTED** [review]
we shouldn't be mixing production code w/ testing code; additionally, the usage of negative margins might be problematic - i might be lacking context here, but best is if we can reduce the actual padding elsewhere rather than using negative margins

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx (hunk 2 above)
Comment (kishore03109) [reply src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:44]: the code is already in main line hahah, just so happens hidden behind feature flag. 
this means the removal of a fair bit of code + then re-reviewal of already reviewed code, which feels counterproductive, thoughts?

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx (hunk 3 above)
Comment (kishore03109) [reply src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:133]: sadly we dont have a token for this yet.
@sehyunidaaa may i get your help in this ya

**Comment by kishore03109** [comment]
test

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx [hunk 5]
```diff
@@ -17,25 +17,31 @@ export const IsomerDetailGroupView = ({
...
+          borderRadius="0.25rem"
+          border="1px solid"
           borderColor="base.divider.medium"
-          display="flex"
-          justifyContent="flex-end"
+          boxShadow="0px 8px 12px 0px rgba(187, 187, 187, 0.50)"
+          px="0.5rem"
+          py="0.25rem"
+          mr="-0.5rem"
+          mt="-1.5rem"
```
[the remark this answers is not in the document]
Comment (seaerchin) [reply src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:34]: hmm i get that - but the odd thing is i couldn't find negative margins being used in card grid ._. 

i'm ok with this for now ba but if there's time, please try to figure out why card grids doesn't require the negative margin

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx [hunk 6]
```diff
@@ -16,70 +14,48 @@ export const IsomerDetailGroupView = ({
...
+        justifyContent="end"
       >
         <HStack
+          bgColor="interaction.mainLight.default"
+          borderRadius="0.25rem"
+          border="1px solid"
           borderColor="base.divider.medium"
-          display="flex"
-          justifyContent="flex-end"
+          boxShadow="0px 8px 12px 0px rgba(187, 187, 187, 0.50)"
```
Comment (seaerchin) [inline src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:30]: could we use a token for this?
```ts
boxShadow: `0 0 0 1px var(--chakra-colors-gray-400)`
```

**Review by seaerchin — APPROVED** [review]
please use colour tokens!

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx (hunk 6 above)
Comment (kishore03109) [reply src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:30]: @sehyunidaaa may i get token for this please?

Comment (kishore03109) [reply src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:30]: can use chakra tokens from here https://github.com/chakra-ui/chakra-ui/blob/f65586674cb9241feeacfce7c5672c9129d1704e/packages/theme/src/foundations/colors.ts#L34

Comment (sehyunidaaa) [reply src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:30]: hmmm can we use this `shadow-sm` from OGPDS? 
https://github.com/opengovsg/design-tokens/blob/09eeb37eaca44f0cb56e64d992cdee75b34e7661/tokens.json#L1109C1-L1110C1 ? might look a bit different but not earth-shattering, it's a small shadow

but we've been using the same bespoke shadow for other bubble menus btw
https://github.com/isomerpages/isomercms-frontend/blob/825766a6fc7aa300ba44a87899b624107c255e31/src/layouts/components/Editor/components/CardsBubbleMenu.tsx#L20

File: src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx (hunk 3 above)
Comment (sehyunidaaa) [reply src/layouts/components/Editor/extensions/Details/IsomerDetailGroupView.tsx:118]: ahh this is cos i used the colors from the template. my bad!

can use `divider/strong` [from OGPDS please](https://github.com/opengovsg/design-tokens/blob/09eeb37eaca44f0cb56e64d992cdee75b34e7661/tokens.json#L640) thank you

**Pull request merged**